bcManager/bcManager.py

```diff
@@ -773,9 +1404,45 @@ def get_replay_teams_and_players(self, replay):
             }
         }
     
-    # TODO: update to lookup request
-    async def get_steam_ids(self, guild, player: discord.Member):    
-        return ['76561198380344413', '76561199064986643']
+    async def send_ram_message(self, channels, embed: discord.Embed):
+        messages = []
+        for channel in channels:
+            if channel:
+                message = await channel.send(embed=embed)
+                messages.append(message)
+        return messages
+
+    async def update_messages(self, messages, embed: discord.Embed):
+        for message in messages:
+            await message.edit(embed=embed)
+
+    def parse_group_code(self, code_or_link: str):
+        if 'group/' in code_or_link:
+            code_or_link = code_or_link.split('group/')[-1]
+        
+        return code_or_link  # returns code
+
+    async def get_player_accounts(self, player: discord.Member, platforms=[]):
+        url =  f"{RSC_WEB_APP}/api/member/{player.id}/accounts"
+        rsc_app_token: str = await self._get_rsc_app_token(player.guild)
+        headers = {"X-Api-Key": rsc_app_token}
+        data = requests.get(url, headers=headers).json()
```

Comment:
would we prefer `aiohttp` or similar async networking library so we can do this w/o blocking?

Reply:
We already use to_thread in a bunch of places, I can update it to be consistent with that.